index.html

```diff
@@ -58,8 +58,8 @@
         <div class="bg-gray-100 rounded-xl md:rounded-3xl mx-4 md:max-w-7xl md:mx-auto px-4 py-16 md:py-32 mt-8 md:mt-16">
             <div class="text-center max-w-3xl mx-auto">
                 <h1 class="text-4xl md:text-7xl font-bold text-gray-900 mb-6 md:mb-8 font-redhat">Never overpay your loan again.</h1>
-                <p class="text-lg md:text-xl font-redhat text-gray-500 max-w-xl mx-auto">Sleep well knowing your Liquity interest rates adjust automatically to market conditions.</p>
-                <button class="bg-secondary font-redhat text-white px-6 py-3 md:py-4 mt-6 md:mt-8 rounded-full text-md w-full md:w-auto">Delegate now</button>
+                <p class="text-lg md:text-xl font-redhat text-gray-500 max-w-xl mx-auto">Sleep well knowing your loan automatically adjusts to market conditions with a low but safe interest rate.</p>
+                <a href="#howitworks"><button class="bg-secondary font-redhat text-white px-6 py-3 md:py-4 mt-6 md:mt-8 rounded-full text-md w-full md:w-auto">Delegate now</button></a>
```

Comment:
I think we can get rid off `<button>` here then

Reply:
I tried that initially but with an `<a>` alone the margin was messed up and I didn't want to mess around. Wrapping it solved it. :man_shrugging:

Reply:
Fair enough. Nobody will notice anyway :D